pkg/utils/workloadidentity/jwt.go

```diff
@@ -27,6 +28,7 @@ const (
 
 var (
 	now = time.Now
+	uid = uuid.NewUUID
```

Comment:
I suggest to use "github.com/google/uuid" directly as you cast the uid to string anyways later on.

```suggestion
	uid = uuid.New
```